docs/HowToInstall.md

```diff
@@ -0,0 +1,83 @@
+# How to install?
```

Comment:
Technically, 'how to install' is noun clause.
Maybe better to use something like the following:

- How to install

or 

- How to install vscode extension for ONE